runtime/syntax/dircolors.vim

```diff
@@ -199,11 +199,11 @@ endfunction
 
 let b:dc_next_index = 0
 
-if has('gui_running')
+if (has('gui_running') || &termguicolors==1)
     call s:set_guicolors()
 endif
 
-if has('gui_running') || &t_Co != ''
+if (has('gui_running') || &termguicolors==1) || &t_Co != ''
```

Comment:
the parens are pointless and confusing here.

Reply:
Well, that was just a blind replace&check.
For now, I looked the file carefully and (kinda) fixed the issue.

What do you say about current state?